src/components/Chatbar/components/Conversation.tsx

```diff
@@ -186,6 +196,35 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
     }
   }, [isRenaming, isDeleting]);
 
+  const handleOpenSharing: MouseEventHandler<HTMLButtonElement> =
```

Comment:
some days ago you said that doing so is antipattern
Something changed?

Reply:
it's used as property of child React component